### PR TITLE
Homework week 1 MUCAHIT

### PR DESCRIPTION
**Pull request by mucahit04**

### CODE REVIEW

File: Week1/homework/animals_array.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+ let animals = [];
+ console.log("the value of animals array is should be 'undefined' ");
```
Comment (unmeshvrije) [inline Week1/homework/animals_array.js]: is it undefined ?

File: Week1/homework/animals_array.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+ console.log(animals);
+ let my_animals = ["cat", "bird", "dolphin"];
+ console.log(my_animals);
+ animals = ["Piglet"];
```
Comment (unmeshvrije) [inline Week1/homework/animals_array.js]: Does this line **add** the element ?

File: Week1/homework/log_remainder.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+/*
+1- If x equals 7, and the only other statement is x = x % 3, what would be the value of x 
+after the calculation ?*/
+
+let x = 7; // this is the decleration and assigning the value of 7 to x.
+x = x % 3; /* This piece of code assigns a new value to the x and that is the remainder of x/3. The remainder of 7/3= 1 */
```
Comment (unmeshvrije) [inline Week1/homework/log_remainder.js]: nice detailed comments!

File: Week1/homework/log_string.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+ let myString = "Mucahit Kahraman";
+ console.log("the value of myString is 'Mucahit Kahraman' ");
+ console.log(myString);
+ myString = "I love Hack Your Future!";
```
Comment (unmeshvrije) [inline Week1/homework/log_string.js]: How nice!

File: Week1/homework/log_the_number.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+ let numberX;
+ console.log("The value of numberX is should be 'undefined!' as far as I remember");
```
Comment (unmeshvrije) [inline Week1/homework/log_the_number.js]: you can also check it by printing and then change your comment :)

File: Week1/homework/round_and_log_number.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+ let z = 7.25;
+ console.log(z);
+ let a = Math.round(z);
+ console.log(a);
+ let highest_number = Math.max(z, a);
```
Comment (unmeshvrije) [inline Week1/homework/round_and_log_number.js]: good use of max()

File: Week1/homework/type_checker.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+}
+if (typeof str2 === typeof bicycle) {
+    console.log("Same type!");
+}
+if (typeof car === typeof bicycle) {
+    console.log("Same type!");
+}
+console.log(typeof str2);
+console.log(typeof car);
+if (typeof str2 != typeof car) {
```
Comment (unmeshvrije) [inline Week1/homework/type_checker.js]: you could (should) just write else case for every if.

File: Week1/homework/animals_array.js (hunk 1 above)
Comment (mucahit04) [reply Week1/homework/animals_array.js]: It is not of course:) I learnt that it is an empty array but I just wanted to keep it with my first answer so it shows that I couldn't make it correct at first.

File: Week1/homework/animals_array.js (hunk 2 above)
Comment (mucahit04) [reply Week1/homework/animals_array.js]: Well it is not adding actually. But practically it makes the array having the element "Piglet" in the array. I should use .push method, right?

File: Week1/homework/log_remainder.js (hunk 3 above)
Comment (mucahit04) [reply Week1/homework/log_remainder.js]: thanks

**Comment by mucahit04** [comment]
> Well done. I like the naming consistency for files. All files should have been inside the js-exercises folder.

Thanks for the compliments. Yes I just realised that I forgot to create js-exercises folder when I was redoing the pull request part. My bad.

File: Week1/homework/log_the_number.js (hunk 5 above)
Comment (mucahit04) [reply Week1/homework/log_the_number.js]: same reason of "I would like to keep first answers here so its like a quiz that I answered".

File: Week1/homework/animals_array.js (hunk 2 above)
Comment (unmeshvrije) [reply Week1/homework/animals_array.js]: yes

File: Week1/homework/animals_array.js (hunk 1 above)
Comment (unmeshvrije) [reply Week1/homework/animals_array.js]: okay. understood

File: Week1/homework/type_checker.js (hunk 7 above)
Comment (mucahit04) [reply Week1/homework/type_checker.js]: Wouldn't it be more messing up code? I mean I tried to go along the requirements of the task for this one and this was the only way popped in my mind.

Comment (unmeshvrije) [reply Week1/homework/type_checker.js]: It's not wrong. but it seems like you already know the answers before writing if/else. And then you write != if condition only to cover the "not equal" case.

**Pull request closed without merging**